content/en/docs/Writing policies/variables.md

```diff
@@ -316,6 +316,41 @@ cm-array-example:
 
 Changing the `role` annotation to one of the values present in the ConfigMap, for example `tenant-admin`, allows the Deployment resource to be created.
 
+### Mixing ConfigMap with AdmissionReview data (Nesting)
```

Comment:
Should we call this "Double Dispatch Lookups"? 

Double Dispatch is a OO code pattern where the behavior varies based on two or more parameters. Here, we are varying policy behavior based on 2 or more variables, hence it seems to apply.

Reply:
I had to look that term up and I still don't fully understand it :) I think the distinction here is how the policy language is written versus its behavior. From the written angle, it seemed like the simplest way to refer to this presentation was using the term "nesting/nested" but also because it's a familiar term to most. Happy to change it to Double Dispatch Lookups, but expect users/readers to not know what that is (like I).

Reply:
Makes sense. In that case suggest calling the title, "Nested lookups". We will soon have additional data sources (API requests) so may be best to leave those out of the title.

Reply:
Yes, I think "Nested Lookups" is probably a better, more generic way to refer to how this logic gets written. Will update.